docs/git-mirror.md

```diff
@@ -0,0 +1,173 @@
+# How Git Mirrors Works
+
+## The goal
+
+The goal of Git mirrors is to:
+
+- reduce network bandwidth, and
+- reduce disk usage
+
+for machines running multiple agents side by side.
```

Comment:
maaaaaaaaybe it's worth talking about how "git mirror on a ZFS share" is also a supported workflow?

Reply:
Congratulations! You have won one (1) footnote!